README.md

```diff
@@ -117,8 +117,10 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Anime & Manga
 
+* [**Aniyomi**](https://github.com/aniyomiorg/aniyomi)
 * [**Bangumi**](https://github.com/czy0729/Bangumi) <sup>**[[F-Droid](https://www.f-droid.org/app/com.czy0729.bangumi)]**</sup>
 * [**Kotatsu**](https://github.com/nv95/Kotatsu) <sup>**[[F-Droid](https://f-droid.org/app/org.koitharu.kotatsu)]**</sup>
+* [**Mihon**](https://github.com/mihonapp/mihon)
```

Comment:
That one seems to include proprietary Google components.

Reply:
Yeah, you are correct, they do use Firebase for crash analytics/reports.
You can check it [here](https://github.com/mihonapp/mihon/blob/main/app/build.gradle.kts).

Reply:
Mihon doesn't meet the inclusion criteria, but Aniyomi seems fine.

Reply:
Done.